app/themes/dashboard/ui/app/root/root.controller.js

```diff
@@ -0,0 +1,108 @@
+(function () {
+  'use strict';
+
+  angular.module('app.root')
+    .controller('FooterCtrl', FooterCtrl)
+    .controller('RootCtrl', RootCtrl);
+
+  function FooterCtrl() {
+    this.currentYear = new Date().getUTCFullYear();
+  }
```

Comment:
Interesting, in 3-column layout currentYear is put onto the rootCtrl. I don't mind either way, but perhaps best to keep it consistent?